### PR TITLE
Problem: missing Makefile equivalent for windows (fix #443)

### LINKED ISSUE #443: Problem: missing Makefile equivalent for windows
Makefile doesnt't work on windows
simple Makefile.bat would be good to compile binaries from sources

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add compiling script for windows
needs mingw64 beforehand for native compile

### CODE REVIEW

**Comment by leejw51crypto** [comment]
image

**Review by tomtau — COMMENTED** [review]
can this be tested on that Windows-host Github action?

**Comment by leejw51crypto** [comment]
yes, it's feasible
https://github.com/marketplace/actions/install-mingw
i'll add

**Review by atyhieu — APPROVED** [review]

**Review by atyhieu — APPROVED** [review]

**Review by atyhieu — APPROVED** [review]

File: .github/workflows/buildwin.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+      - name: show info
+        run: |
+          echo "Working Folder $Env:GITHUB_WORKSPACE"
+      - name: buildwindows
+        run: |
+          cd $Env:GITHUB_WORKSPACE
+          cd
+          dir
+          makewin.bat
```
Comment (tomtau) [inline .github/workflows/buildwin.yml:33]: maybe you need to call `cmd makewin.bat`

**Review by ytbigvinny288 — APPROVED** [review]

**Review by ytbigvinny288 — APPROVED** [review]

**Review by ytbigvinny288 — APPROVED** [review]

**Review by ytbigvinny288 — APPROVED** [review]

**Review by tomtau — APPROVED** [review]

**Review by ytbigvinny288 — COMMENTED** [review]
🥸

**Review by ytbigvinny288 — COMMENTED** [review]
than

**Pull request merged**